spec/requests/lettings_logs_controller_spec.rb

```diff
@@ -333,6 +333,14 @@
           end
 
           context "with year filter" do
+            before do
+              Timecop.freeze(2022, 3, 1)
```

Comment:
personally i would just change all these to use `around` and switch to the block syntax, maybe that a preference thing though

Reply:
some of these are necessary but have changed the ones that can be